tox.ini

```diff
@@ -73,3 +75,20 @@ commands = pdm export \
 [testenv:pdm-lock-check]
 allowlist_externals = pdm
 commands = pdm lock --check
+
+[testenv:hadolint]
+allowlist_externals = hadolint
+groups = dev
+commands = hadolint -V --failure-threshold warning \
+           --info DL3006 --info DL3007 --info DL3013 --info DL3041 \
```

Comment:
Can you please describe what are those `DL` rules and why we need to specifically make them as info?

Reply:
These DL all relate to pinning versions in different package managers. See [https://github.com/hadolint/hadolint/wiki/DL3006](DL3006: include image tag), [https://github.com/hadolint/hadolint/wiki/DL3007](DL3007: warning against using `:latest` tag), [DL3013: Pip package version pinning] (https://github.com/hadolint/hadolint/wiki/DL3013) and [DL3041: dnf version pinning](https://github.com/hadolint/hadolint/wiki/DL3041). I wanted to keep the messages there as a *best practice*, but did not want to fail the validation because I feel like using latest images and packages is the entire purpose of building images on every push.

Shall I:
- Actually pin the versions
- Write comments describing why this step was done or
- Leave as-as?

Reply:
When I tried to run on my computer I only see `DL3041` and `DL3013` as a warning. I am fine with keeping those as indo, but are you sure other are also needed?

Reply:
You are correct, they aren't actually needed. I will remove them.